model/project_configs_db.go

```diff
@@ -28,6 +28,12 @@ var (
 	ProjectConfigWorkstationConfigKey      = bsonutil.MustHaveTag(ProjectConfig{}, "WorkstationConfig")
 	ProjectConfigCommitQueueKey            = bsonutil.MustHaveTag(ProjectConfig{}, "CommitQueue")
 	ProjectConfigTaskSyncKey               = bsonutil.MustHaveTag(ProjectConfig{}, "TaskSync")
+	ProjectConfigPRTestingEnabledKey       = bsonutil.MustHaveTag(ProjectConfig{}, "PRTestingEnabled")
```

Comment:
There's no reason to keep adding these if we aren't using them (I feel like we were at some point for updates but i'm not seeing that they have a use now).